tests/apps/sequencing_metrics_parser/parsers/test_bcl_convert_to_sequencing_statistics.py

```diff
@@ -32,4 +33,37 @@ def test_create_sequencing_statistics_from_bcl_convert_metrics(
         == len(parsed_bcl_convert_metrics.get_sample_internal_ids()) * 2
     )
 
-    # THEN assert that the sequencing statistics are correct
+
+def test_create_undetermined_sequencing_statistics_from_bcl_convert_metrics(
+    bcl_convert_metrics_dir_path: Path,
+):
+    """Test creating undetermined sequencing statistics from bcl convert metrics."""
+
+    # GIVEN a directory with a flow cell demultiplexed with bcl convert with undetermined reads
+
+    # WHEN creating undetermined sequencing statistics from bcl convert metrics
+    metrics: List[SampleLaneSequencingMetrics] = create_bcl_convert_undetermined_metrics(
+        flow_cell_dir=bcl_convert_metrics_dir_path,
+        non_pooled_lane_sample_pairs=[(1, "sample_id")],
+    )
+
+    # THEN metrics are created for the undetermined reads
+    assert isinstance(metrics, list)
+    assert isinstance(metrics[0], SampleLaneSequencingMetrics)
+
+
+def test_create_undetermined_sequencing_statistics_from_bcl_convert_metrics_for_existing_lane_without_undetermined_reads(
+    bcl_convert_metrics_dir_path: Path,
+):
+    """Test creating undetermined sequencing statistics from bcl convert metrics."""
```

Comment:
Feel free to adjust it so that it doesn't exceed 100 characters long
```suggestion
    """Test creating undetermined sequencing statistics from bcl convert metrics without undetermined reads."""
```